src/app/pages/landingpage/landingpage.component.html

```
@@ -9,36 +9,41 @@
+              <button
+                type="button"
+                class="inline-flex min-h-[2.5rem] w-full items-center justify-center bg-transparent px-5 py-2 text-center font-bold leading-snug text-teal-500 hover:text-teal-600"
+                (click)="scrollTo('resource-tracker')"
               >
-                <button
-                  class="bg-transparent text-teal-500 py-5 px-5 font-bold hover:text-teal-600"
-                >
-                  Learn more >
-                </button>
-              </a>
+                Measure AI/ML jobs >
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)